tests/platform_tests/api/platform_api_test_base.py

```diff
@@ -0,0 +1,32 @@
+from common.helpers.assertions import pytest_assert
+
+
+class PlatformApiTestBase(object):
+    """Platform API test base class"""
+
+    failed_expectations = []
+
+    def expect(self, expr, err_msg):
+        """
+        A pytest method can call this method multiple times. It will accumulate
+        error messages for each expression which would have failed an
+        assertion in failed_expectations. When the test method is ready to
+        check if any of the previous expressions passed to this function would
+        have failed an assertion, it should call assert_expectations()
+        """
+        if not expr:
+            self.failed_expectations.append(err_msg)
+        return expr
+
+    def assert_expectations(self):
```

Comment:
make this a shared helper?

Reply:
never mind. you have specific things.

Reply:
I considered it. We might be able to move it to common/ in the future?